app/meetings/migrations/0001_initial.py

```diff
@@ -1,8 +1,8 @@
 # Generated by Django 2.2.6 on 2020-01-15 21:55
 
+import django.db.models.deletion
```

Comment:
Why to change the order? It was alphabetical. Isort conf needs fixing?

Reply:
isort divides import into groups:
python-internal
third party
own
and then in the groups all imports are sorted alphabetically